src/main/resources/META-INF/plugin.xml

```diff
@@ -2,7 +2,7 @@
     <id>de.php_perfect.intellij.ddev</id>
     <name>DDEV Integration</name>
     <version>1.0.0</version>
-    <vendor email="nico@php-perfect.de" url="https://github.com/php-perfect">PHP-Perfect</vendor>
+    <vendor email="dev@akiba.at" url="https://github.com/ddev/ddev-intellij-plugin">DDEV IntelliJ Plugin</vendor>
```

Comment:
Let's use `ddev-intellij-plugin@ddev.com` 

```suggestion
    <vendor email="ddev-intellij-plugin@ddev.com" url="https://github.com/ddev/ddev-intellij-plugin">DDEV IntelliJ Plugin</vendor>
```

Or consider using `accounts@ddev.com` which is simpler, but it would mean we'd ping you.

Reply:
If you'd like the separate email address, we'll make a google group that forwards to all including you and probably nico too.